python/MaterialXTest/genMaterialsFromDefs.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python
+'''
+Utility to create a document per node definition such that an instance of that 
```

Comment:
This seems more like a pipeline-specific utility script, rather than a component of our standard Python test suite.  Maybe this should be omitted from the pull request, and we can consider individual elements of this for inclusion further in the future?

Reply:
True enough. Script has been removed.